agimus_demos_common/agimus_demos_common/launch_utils.py

```diff
@@ -97,7 +97,7 @@ def generate_include_franka_launch(launch_file_name: str) -> IncludeLaunchDescri
     )
 
 
-def get_use_sime_time() -> dict[str, LaunchConfiguration]:
+def get_use_sim_time() -> dict[str, LaunchConfiguration]:
```

Comment:
Is there any way this can break backward compatibility?
If yes, you can add
```python
get_use_sime_time = get_use_sim_time
```
or something more elaborated that will warn the user.

Reply:
Done